app/locales/fr.json

```diff
@@ -0,0 +1,99 @@
+    "location_disabled_title": "Le suivi de la position a été désactivé",
+    "location_enabled_message": "COVID Safe Paths enregistre de façon sécurisée vos coordonnées GPS toutes les cinq minutes sur cet appareil..",
+    "location_enabled_title": "COVID Safe Paths activé",
+    "maps_import_button_text": "Importer les positions passées",
+    "maps_import_disclaimer": "L’application Safe Paths n’est pas affiliée à Google et ne partage jamais vos données.",
+    "maps_import_text": "Vous pouvez importer votre historique personnel de position pour voir si vous avez croisé une personne infectée avant d’avoir téléchargé cette application.",
+    "maps_import_title": "Google Maps",
+    "nCoV2019_url_info": "Plus d’informations sur les données représentées sur cette carte",
+    "news_subtitle": "Obtenez les dernières nouvelles sur le COVID-19 par vos autorités sanitaires et en général.",
+    "news_title": "Dernières nouvelles",
+    "no_data": "Pas de données",
+    "notification_2_weeks_ago": "Il y a 2 semaines",
+    "notification_data_not_available": "Pas de données d’exposition disponibles.",
+    "notification_random_data_button": "Choisir une autorité sanitaire",
+    "notification_title": "Profil des expositions sur 2 semaines",
+    "notification_today": "aujourd’hui",
+    "notification_warning_text": "Si une autorité sanitaire dans votre région a mis en place Safe Paths vous pouvez abonner pour obtenir des mises à jours régulières sur les risques d’exposition.",
+    "notifications_exposure_format": "Vous avez croisé une personne infectée il y a {{daysAgo}} jours pendant {{exposureTime}} minutes.",
+    "notifications_exposure_format_today": "Vous avez croisé une personne infectée aujourd’hui pendant {{exposureTime}} minutes.",
+    "notifications_exposure_format_yesterday": "Vous avez croisé une personne infectée hier pendant {{exposureTime}} minutes.",
+    "notifications_no_exposure": "Pas d’expositions connues à la COVID-19 durant les deux dernières semaines.",
+    "overlap_found_button_label": "Données publiques chargées",
+    "overlap_no_results_button_label": "Données publiques chargées",
+    "overlap_para_1": "Le chemin vert représente votre historique de positions\n\nLes cercles violets représentes les données publiques",
+    "overlap_title": "Chercher les recoupements",
+    "push_at_risk_message": "Vous avez croisé une personne infectée à la COVID-19",
+    "push_at_risk_title": "Vous avez pu être exposé",
+    "settings_title": "Réglages",
+    "share_location_data": "Partager les données de position",
+    "show_overlap": "Cliquez pour voir les données publiques",
+    "team": "L’équipe",
+    "team_para": "Notre équipe est un consortium composé d’épidémiologistes, d’ingénieur·es, de scientifiques des données, de défenseurs de la vie privée numérique, de professeur·es et de chercheur·ses d’institutions réputées: dont: Le MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young, et Link Ventures.",
```

Comment:
institutions réputées, dont le MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young et Link Ventures.